tests/ipv6-evasion/ipv6-dos-with-ext-headers-5/README.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+Test detection of DoS attack that tries to increase decoding effort.
+
+# PCAP
+
+Pcap from https://redmine.cs.uni-potsdam.de/projects/pcap/files
+
+# Notes
+
+Currently skipped as no rule available yet.
+
```

Comment:
I am not sure what is expected here
react to a huge number of ICMPv6 pings without answers ?

or to the packets with the Wireshark filter `icmpv6.type == 4`  parameter problem (unrecognized next header type)
These packets seem decoded fine by Wireshark, and not by the target...

Reply:
So far I have no good idea how to detect it, at least they didn't find a solution at the paper as well.